kernel/data-pipeline/cdc/protocol/src/main/proto/CDCRequestProtocol.proto

```diff
@@ -27,24 +27,24 @@ message CDCRequest {
   enum Type {
     UNKNOWN = 0;
     LOGIN = 1;
-    CREATE_SUBSCRIPTION = 2;
-    START_SUBSCRIPTION = 3;
-    ACK = 4;
-    STOP_SUBSCRIPTION = 5;
-    DROP_SUBSCRIPTION = 6;
+    STREAM_DATA = 2;
+    START_STREAMING = 3;
+    ACK_STREAMING = 4;
+    STOP_STREAMING = 5;
+    DROP_STREAMING = 6;
   }
   Type type = 3;
-  oneof request {
-    LoginRequest login = 4;
-    CreateSubscriptionRequest create_subscription = 5;
-    StartSubscriptionRequest start_subscription = 6;
-    AckRequest ack_request = 7;
-    StopSubscriptionRequest stop_subscription = 8;
-    DropSubscriptionRequest drop_subscription = 9;
+  oneof request_body {
+    LoginRequestBody login_request_body = 4;
+    StreamDataRequestBody stream_data_request_body = 5;
+    StartStreamingRequestBody start_streaming_request_body =6;
```

Comment:
1, Move `StartStreamingRequestBody` after `StopStreamingRequestBody`, since it just be used after stopping.

2, ` =6;` could be ` = 6;`